spec/models/card_spec.rb

```diff
@@ -0,0 +1,25 @@
+require 'rails_helper'
+
+describe Card do
+  it 'has a valid factory' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.